style.css

```diff
@@ -0,0 +1,598 @@
+* {
+  --main-bg-color: #fff;
+  --text-color: #6070ff;
+  --links-color: #344563;
+  --intro-color: #091e42;
+  --primary-B300: #7f8cff;
+  --icon-color: #505f79;
+  --body-color: #e5e5e5;
+  --desc-color: #091e42;
+  --experience: #6b778c;
+  --main-font-family: 'Poppins';
+  --footer-background: #6070ff;
```

Comment:
Please make sure to use the corresponding font for your project. To use custom fonts, you will need to first add them to your project. The easiest way to do it is by adding the font to your HTML `head`. I will explain to you a way of doing this using [Google Fonts](https://fonts.google.com/) 😄: 

- Search for the font that you want: 
![image](https://user-images.githubusercontent.com/42799579/198803592-001b8e4a-8f09-40bb-b094-d8196a4bcdab.png) 

- Select it: 
![image](https://user-images.githubusercontent.com/42799579/198803724-bb8ef580-55e5-454c-a351-25f326eb2992.png) 

- You will be taken to the font page, scroll down until you have the weights: 
![image](https://user-images.githubusercontent.com/42799579/198804057-63e483f4-fc4e-469b-b9ac-cf6d611be885.png) 

- Select the ones that you want by clicking here: 
![image](https://user-images.githubusercontent.com/42799579/198805635-c8e31d9e-82f6-4c72-8bdf-ffe170f5683f.png) 

- A box will appear on the right, after selecting all the fonts that you want, copy the link: 
![image](https://user-images.githubusercontent.com/42799579/198804518-6865e9ea-f5da-4636-9778-a7b3f6bc12a2.png) 

And add the copied link to your HTML `head` 😉. 
> Take into account that you can select multiple fonts & weights 😃